datalad/tests/test_utils.py

```diff
@@ -1139,6 +1140,8 @@ def f(j):
 
 @with_tempfile(mkdir=True)
 def test_dlabspath(path):
+    if not has_symlink_capability():
```

Comment:
Ideally we should just make it into a decorator to stay consistent?

Reply:
I have mixed feelings re decorators. They definitely shouldn't be the only method. I have RF'ed so many tests recently that could run almost in full under all conditions, but had a single conditional that would have needed scenario-specific treatment, but only a decorator was available, hence I had to disable the full test...